src/diffusers/pipelines/cogvideo/pipeline_cogvideox.py

```diff
@@ -481,9 +486,9 @@ def __call__(
         self,
         prompt: Optional[Union[str, List[str]]] = None,
         negative_prompt: Optional[Union[str, List[str]]] = None,
-        height: int = 480,
-        width: int = 720,
-        num_frames: int = 49,
+        height: Optional[int] = None,
```

Comment:
This is not a breaking change since we can determine these exact values using transformer config parameters